tests/pivots/spreadsheet_pivot/spreadsheet_pivot_side_panel.test.ts

```diff
@@ -134,4 +135,33 @@ describe("Spreadsheet pivot side panel", () => {
     await click(fixture.querySelector(".fa-undo")!);
     expect(fixture.querySelectorAll(".pivot-dimension")).toHaveLength(0);
   });
+
+  test("should preserve the sorting of the dimension after ordering is changed", async () => {
+    mockGetBoundingClientRect({
+      "pt-1": () => ({
```

Comment:
maybe a comment/a more precise selector here ? It's kinda hard to understand what `pt-0` is when reading the test :x